src/Foundation.Blazor.Client/Pages/Patient.razor

```diff
@@ -75,43 +75,67 @@
     private string DialogTitle = string.Empty;
     private bool IsNew = true;
     private List<object> ToolbarItems = new();
-    private string[] SpecificColumns = ["PatientNumber"];
+    private string[] SpecificColumns = ["PatientNumber", "Name"];
+    private string SearchValue = string.Empty;
 
     [Parameter]
-    [SupplyParameterFromQuery]
+    [SupplyParameterFromQuery] 
     public string DoctorId { get; set; }
 
     private DateTime? SelectedDate
     {
         get => CurrentPatient.DateOfBirth == default
-            ? DateTime.Today
-            : CurrentPatient.DateOfBirth.ToDateTime(TimeOnly.MinValue);
+        ? DateTime.Today
+        : CurrentPatient.DateOfBirth.ToDateTime(TimeOnly.MinValue);
 
         set => CurrentPatient.DateOfBirth = value.HasValue
-            ? DateOnly.FromDateTime(value.Value)
-            : default;
+        ? DateOnly.FromDateTime(value.Value)
+        : default;
     }
 
     protected override async Task OnInitializedAsync()
     {
-        await LoadPatients();
-
         ToolbarItems.AddRange(new object[]
         {
-            new ItemModel() { TabIndex=0 , Text = L["Add"], TooltipText = L["Add"], PrefixIcon = "e-add" },
+            new ItemModel() { TabIndex = 0, Text = L["Add"], TooltipText = L["Add"], PrefixIcon = "e-add" },
             new ItemModel() { TabIndex = 1, Text = L["Edit"], TooltipText = L["Edit"], PrefixIcon = "e-edit" },
             new ItemModel() { TabIndex = 2, Text = L["Delete"], TooltipText = L["Delete"], PrefixIcon = "e-delete" },
             new ItemModel() { TabIndex = 3, Text = L["Refresh"], TooltipText = L["Refresh"], PrefixIcon = "e-refresh" },
             (object)"Search"
         });
     }
 
+    public async Task ActionBeginHandler(ActionEventArgs<PatientDto> args)
+    {
+        if (args.RequestType == Syncfusion.Blazor.Grids.Action.Searching)
+        {
+            args.Cancel = true; // cancel default searching
```

Comment:
@ssag26 avoid inline comment,  document above the method signature will be better